compliance/cis_k8s/rules/cis_4_2_12/data.yaml

```diff
@@ -1,6 +1,6 @@
 metadata:
   id: 46d69440-3946-50f6-83b3-a0987551afa2
-  name: Verify that the RotateKubeletServerCertificate argument is set to true (Manual)
+  name: Verify that the RotateKubeletServerCertificate argument is set to true or it's not present (Manual)
```

Comment:
please don't change that as this is thew official name of the rule by CIS and not our own name